### PR TITLE
Bump istio dependencies

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

https://github.com/kmesh-net/kmesh/pull/927
https://github.com/kmesh-net/kmesh/pull/928   failure  have some relationship with istio

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: test/e2e/baseline_test.go [hunk 1]
```diff
@@ -1,6 +1,3 @@
-//go:build integ
-// +build integ
```
Comment (hzxuzhonghu) [inline test/e2e/baseline_test.go:2]: cc @YaoZengzeng why did we need this tag

Comment (YaoZengzeng) [reply test/e2e/baseline_test.go:2]: It is used to seperate test files, we will use it in: https://github.com/kmesh-net/kmesh/blob/main/test/e2e/run_test.sh#L278 .

Although all the files in `./test/e2e/` are used to e2e test.

Comment (hzxuzhonghu) [reply test/e2e/baseline_test.go:2]: Will revert

**Pull request merged**